p1.java

```diff
@@ -0,0 +1,37 @@
+// Time Complexity : O(n)
```

Comment:
Check oce more it should be n^2

Reply:
Yes it should be n^2. I will correct it. Thanks!